src/format.ts

```diff
@@ -1,13 +1,31 @@
 import type { MessageEntity } from "./deps.deno.ts";
 
+/**
+ * Objects that implement this interface implements a `.toString()` 
+ * method that returns a `string` value that represents the object.
```

Comment:
```suggestion
 * method that returns a `string` value representing the object.
```

IMO repeating "that" makes it a. bit confusing, specially for non-native english speakers :)